cfc-testing/scripts/deploy-interactive.sh

```
@@ -56,7 +56,7 @@ echo -e "${YELLOW}🚀 Starting interactive deployment...${NC}"
 echo ""
 
 # Run with proper input handling and force interactive mode
-java -cp "target/classes:target/dependency/*" com.cloudforgeci.samples.app.InteractiveDeployer --interactive "$@"
+java -cp "target/classes:target/dependency/*" com.cloudforgeci.samples.app.InteractiveDeployer"$@"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)